railties/lib/rails/generators/rails/credentials/templates/credentials.yml.tt

```diff
@@ -0,0 +1,6 @@
+# aws:
+#   access_key_id: 123
+#   secret_access_key: 345
+
+# Used as the base secret for all MessageVerifiers in Rails, including the one protecting cookies.
+secret_key_base: <%= @secret_key_base %>
```

Comment:
I added `@secret_key_base` so that custom templates could use it without knowledge of how it is generated.  That way, if Rails changes how it is generated in the future, custom templates can automatically benefit.  Custom templates can still choose to ignore `@secret_key_base`, and generate a `secret_key_base` in their own way.

Reply:
I've changed it to a helper method. :+1: